docs/js-core-3/week-3/lesson.md

```diff
@@ -539,46 +539,89 @@ The project is called [TV Show](../tv-show-dom-project/readme.md)
 
 Trainees should group into teams based on the Level that they are working on currently.
 
-## PD (Agile: Scrum planning)
+## PD (Resilience)
 
-**Session objective:** This session introduces roles, processes and ceremonies in a scrum team. 
+**Session objective:** This workshop is about understanding resilience, the challenge of looking for jobs and how you can develop a resilient mindset when being faced with job rejections. At the end of this session, you will be able to understand what resilience is, identify current coping habits, and develop resilience strategies.
```

Comment:
```suggestion
**Session objective:** This workshop is about understanding resilience, the challenge of looking for jobs and how you can develop a resilient mindset when being faced with job rejections. At the end of this session, you will be able to explain what resilience is, identify current coping habits, and develop resilience strategies.
```

We tend to avoid "understand" in learning objectives, as it's not a very testable goal :)